pkg/volume/local/local.go

```diff
@@ -412,7 +412,7 @@ func (plugin *localVolumePlugin) NodeExpand(resizeOptions volume.NodeResizeOptio
 	case hostutil.FileTypeDirectory:
 		// if the given local volume path is of already filesystem directory, return directly because
 		// we do not want to prevent mount operation from succeeding.
-		klog.InfoS("expansion of directory based local volumes is NO-OP", "local-volume-path", localDevicePath)
+		klog.InfoS("Expansion of directory based local volumes is NO-OP", "local-volume-path", localDevicePath)
 		return true, nil
 	default:
 		return false, fmt.Errorf("only directory and block device are supported")
```

Comment:
should be capitalized for this line?

Reply:
According to https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Capitalize the first letter of the sentence